### ISSUE TITLE
New Benzoxazole Samples Incoming

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMycetoma/series-5-benzimidazoles#3]
@bebi78 asked @bendndi whether any samples of benzoxazoles (structurally related to this series) might be available from DNDi's other projects.

Before launching into any major synthesis we'd really like to explore other, existing, members of the series to make sure we're onto something useful. While we want to prioritise novel structures with novel MoA, we do need to do our due diligence and follow up on promising hits like these, when/if we can ([recent paper](https://journals.plos.org/plosntds/article?id=10.1371/journal.pntd.0010159)).

Ben suggested that Charlotte Sutherell (Imperial) and Andrea Vernall (Otago) would be the most likely to have samples of Benzoxazoles in this series ready to go, and might be interested in shipping them out to @wwjvdsande for testing. Ben offered to try to cover the shipping costs if not too expensive (Dunedin to Rotterdam is quite a way....). Alternatively they could be shipped to Sydney and included in existing shipments, but let's see how we go @fantasy121 @kym834 

Andrea replied "We have 17 benzoxazoles we can send, 12 (out of 13) that we have samples remaining from recent 2022 synthesis, and 5 of the 2021 compounds we have remaining stored as high purity solids. So a total of 17, see attached.

Happy to post wherever, just let me know, I can weigh out 2-3 mg into a smaller vial."

These samples derive from [this paper](https://pubs.acs.org/doi/full/10.1021/acs.jchemed.1c01213).

Files:

[Datawarrior Files.zip](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/files/8908211/Datawarrior.Files.zip)

![IMG_3831](https://user-images.githubusercontent.com/4386101/173801169-d3bd2a48-bf11-476f-9862-0ade64ab0ec7.jpeg)

@bebi78 also mentioned that Enamine might have such compounds or libraries in stock. Some companies of the DNDi NTD Drug Discovery Booster Hit-to-Lead program, e.g., Takeda and Daichii-Sankyo, have published or patented various benzoxazoles as kinase inhibitors or antifungals. Booster S05 project also deals with some interesting azabenzimidazoles. Maybe they can share some of their compounds for mycetoma testing.

Andrea to get a Github account.

- [ ] @bebi78 could you please take the datawarrior files that Andrea shared and convert to a Chemdraw and image file so that we can see the structures that are involved here, and their similarity to the hits we have?

Can be discussed further in OpenSourceMycetoma/Series-1-Fenarimols#77

**Comment by bebi78** [comment]
@mattodd Sorry, can't open that datawarrior files either. Any advise in the group?

**Comment by AndreaOtago** [comment]
Hi everyone! This is Andrea signed up on GitHub. Let me know the postal address/contact etc you want the compounds sent to. I can post a chemdraw/PDF of the structures over the next few days.

**Comment by bendndi** [comment]
@mattodd  @bebi78 Not sure why you can't open that datawarrior file, it works for me.
Unless you don't have datawarrior (!!!!), in which case I immediately suggest you remedy that asap since its the greatest piece of free software ever created, and almost inconceivable to do medchem without it
(https://openmolecules.org/datawarrior/ )

**Comment by bendndi** [comment]
![image](https://user-images.githubusercontent.com/36627399/174294669-da8f3e0c-781e-439b-9d29-79a90e4e6e08.png)

**Comment by bendndi** [comment]
![image](https://user-images.githubusercontent.com/36627399/174294822-5937e433-7ee5-4a9d-b333-8457eb698a57.png)

**Comment by mattodd** [comment]
@bendndi it is indeed fantastic. I was delegating. :)

**Comment by AndreaOtago** [comment]
@mattodd  @bendndi  - let me know what address you want the Otago-made compounds shipped to and I can get that sent away. Shout out to the University of Otago CHEM 305 Undergrad students who made the compounds !

**Comment by wwjvdsande** [comment]
> @[AndreaOtago](https://github.com/AndreaOtago) you can ship it to:
ErasmusMC, department of Medical Microbiology and Infectious Diseases, room NB920
T.a.v. Wendy van de Sande and Ma Jingyi
Westzeedijk 353
3015 AA Rotterdam
The Netherlands

**Comment by AndreaOtago** [comment]
@wwjvdsande - great, I'll get these compounds away next week, and let you know when on the way. Can you please also let me know an email addresses and contact phone number (e.g. good arrival/store is fine), these are required for the shipping paperwork.

**Comment by wwjvdsande** [comment]
@AndreaOtago I just sent you an e-mail with all the contactinformation in it. Might be easier that way so that you can reply or copy paste.

**Comment by AndreaOtago** [comment]
> @AndreaOtago I just sent you an e-mail with all the contactinformation in it. Might be easier that way so that you can reply or copy paste.
![IMG_3841](https://user-images.githubusercontent.com/107620079/177075730-859bd489-69d9-4942-a1bd-be1772cb72d7.jpeg)

Great! Compounds packaged up, will be collected tomorrow, I'll send you the tracking link once I have it

**Comment by mattodd** [comment]
Hi all - have these samples been entered into the Master List with MyOS codes? @AndreaOtago might you be willing to do this? The instructions are pretty simple and are [here](https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention). I'm sure @fantasy121 and @dmitrij176 can assist if there are issues.

**Comment by AndreaOtago** [comment]
Hi @mattodd , I haven't given the compounds MyOS codes, the spreadsheet I sent Wendy with all the coding details has the DNDi code assigned by Ben (and aligned with the 'UO' code the students used on the vials, for their experiments etc). If someone else can help with this that would be great. Is there somewhere to upload an excel sheet or dwr file on here with the DNDi codes etc?

**Comment by csuth22** [comment]
Hi @wwjvdsande, are you still interested in receiving more of similar series to the compounds @AndreaOtago has sent? We've also had students working on these. If yes, I can get these weighed and shipped in the next couple of days - presumably to the same Rotterdam address as above? 

@AndreaOtago did you find a solution to your naming/upload question? 

Best wishes, Charlotte 
  
[Uploading Compounds_Imperial.sdf.zip…]()

**Comment by wwjvdsande** [comment]
@csuth22 yes that would be great. You can use the same address as above. I will be on holiday soon however @MA-Jjingyi will be there when I'm away. She can take care of them and make sure they enter the testing cycle.

**Comment by mattodd** [comment]
Hi @csuth22 - this is absolutely great, thank you! I'm not seeing the file you're uploading - when I click on it nothing happens. Could you please try again? An Excel sheet should work without needing to be zipped. You can also upload images if you want to get across the idea quickly.

For these compounds, and for those from @AndreaOtago we need to assign MyOS codes so that we keep all the data in order, and the compounds need to go into the Master List for the project. The instructions are pretty easy (linked above) and just involve assigning a code (incremental number) along with the other standard data. If you need help, perhaps @dmitrij176 or @fantasy121 or another volunteer data champion can assist with entering the data? But we need it entered I'm afraid.

**Comment by csuth22** [comment]
@wwjvdsande and @MA-Jjingyi that's great, thank you both. If you could let me know an e-mail address and contact details, which I need for shipping information that would be very helpful. If preferred can use c.sutherell@ic.ac.uk to share this.  

@mattodd thanks for letting me know about the format issue; I'm away from my files atm but will upload a dwar file later today

**Comment by AndreaOtago** [comment]
@mattodd @csuth22 I didn't know about the coding before I sent the compounds, so just sent with the DNDi codes as compound ID. I haven't had time to look into coding/re-coding/dual numbering, so hopefully someone else is able to :)

**Comment by csuth22** [comment]
@mattodd thanks for the guide on the naming. All entered into the list with names; not sure if you are formally considering these as series 5 or want a different category; I can update if the latter to preferred name. [Mycetoma Imperial OSN P5 compounds April 2022 (002) (1).csv](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/files/9193111/Mycetoma.Imperial.OSN.P5.compounds.April.2022.002.1.csv)

**Comment by mattodd** [comment]
Thank you @csuth22 - yes we can consider these as Series 5 despite the small differences. Could someone else please help @AndreaOtago enter those compounds?
Cheminformatics specialists like @cdsouthan and @drc007 tell us that we ought also to be filling in the InChI and InChIKey columns in the spreadsheet. If people have bandwidth, please add those in too (columns Q and R).

**Comment by cdsouthan** [comment]
I think the esteemed (other) Chris would support the belt-and-braces mode
of  SMILES + InChI + InChIKey  maybe even full IUPAC as well if Chemdraw
spawns these@

Mat,  going on like a broken record  but you really ought to invest in
https://www.collaborativedrug.com/  for all your collaborative projects

N.b, unlike Labtrove it can be an ELN that will remain accessible :)

COI I am on the AB

[image: image.png]

**Comment by bebi78** [comment]
Just a brief note for all group members that @AndreaOtago and @csuth22 plan to prepare more benzoxazole derivatives in the coming weeks and months.

**Comment by AndreaOtago** [comment]
Compounds incoming! CHEM 305 third year chemistry students at the University of Otago have made 11 benzoxazole deriviaives, datawarrior file attached and a pretty picture showing the different colour of the derivatives. We are sending one set into Mycetoma and one to DNDi OSN project 5 for T.Cruzi and Leishmaniasis (+ panel) testing. DNDi codes tba from @dosreic . What Mycetoma coding is required? 

![CHEM305_2023](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/assets/107620079/9d3532f3-8836-4516-8afe-85c4fe3fff22)

[Otago_cpds_May2023.dwar.zip](https://github.com/OpenSourceMycetoma/series-5-benzimidazoles/files/11528052/Otago_cpds_May2023.dwar.zip)

**Comment by mattodd** [comment]
Awesome @AndreaOtago.
Admin:
1) Compounds need adding to the Master List at http://tinyurl.com/MycetomaMols (let me know if you need permissions)
2) ... for which they need MycetOS codes, which are pretty simple but are described here: https://github.com/OpenSourceMycetoma/MycetOS-Tech-Ops/wiki/The-Molecule-Numbering-Convention
That's it.

**Comment by mattodd** [comment]
Oh, and @AndreaOtago if there were some way we could share your experimental data (NMRs etc) let us know. i.e. would be very happy to host it on one of the project's electronic lab notebooks, if you're amenable (we'd link to it from here). We try to share raw data associated with all molecules, but your data, so your choice!

**Comment by AndreaOtago** [comment]
@mattodd great, thanks. Looks like I have permission to enter new compounds in the sheet. I will wait until the compounds have DNDi codes assigned by @dosreic , then I can enter all the info in one go. I will post the compounds to @wwjvdsande once all this info is in the system.

**Comment by AndreaOtago** [comment]
> 

Yes more than happy to share data, the students have 'signed away' any personal IP as part of the IP form we (the uni) and each student who takes part signs 'with' DNDi, so this translates to any open platform. I am a bit time poor for the next 3-4 weeks in particular but happy to spend time making this work after that.

**Comment by bebi78** [comment]
@AndreaOtago Have you thought of using Lawesson's reagent to thionate your amide compounds?

**Comment by AndreaOtago** [comment]
> @AndreaOtago Have you thought of using Lawesson's reagent to thionate your amide compounds?

Hi @bebi78 , no I haven't, we have 3rd year undergraduates making the compounds and this is not something I would include as part of this undergraduate lab. But yes this is something a postgraduate research student could do if there was interest in making these analogues, I am not sure if OSN Project 5 has any data/compounds with derivitives such as you are suggesting.

**Comment by AndreaOtago** [comment]
Compounds planned to be synthesis by students at Otago in April-May 2025 are posted below, then sent to @wwjvdsande 's lab for testing. This series of compounds has been designed based on SAR to date from series 5b, to fill in some SAR gaps / further develop promising functionality. Note only UOcodes for now, all successfully synthesised compounds will be entered into the MYOS sheet and assigned a MYOS code once ready to send in. 

![Image](https://github.com/user-attachments/assets/90d8aaf8-3074-4c22-9034-01b6686db7c6)

**Comment by AndreaOtago** [comment]
@wwjvdsande compounds incoming! The CHEM 305 students successfully synthesised and purified 10 of the target compounds, structures and a picture below. I will enter the compounds into the MYOS google sheet and assign a MYOS code to all 10 compounds we have to send in. @wwjvdsande is the shipping address the same as last year, who should I address the parcel to c/o what person? 

![Image](https://github.com/user-attachments/assets/4f7138ff-8a4b-471f-a795-a8e5dfe783bc)

![Image](https://github.com/user-attachments/assets/296bac5b-b48e-4863-a8dd-7134688268bd)

**Comment by AndreaOtago** [comment]
@wwjvdsande @mattodd here is the (compressed) data warrior file for the above 10 compounds 

[CHEM305_2025_10compounds.dwar.zip](https://github.com/user-attachments/files/20319975/CHEM305_2025_10compounds.dwar.zip)
